README.md

```diff
@@ -38,10 +38,23 @@ frontend assets are loaded and no javascript or watchdog errors are triggered.
 * Add the provided js file to your sites js. The listener is required to catch js errors.
 
 ### Detecting watchdog errors
+##### Default behaviour
 
 When DrupalErrorCheckApiContext is added, all watchdog entries excluding `Notice`, `Info` and `Debug` will be detected
 and trigger PHP errors in the behat PHP runner automatically.
 
+##### Adjusting watchdog entries severity
+
+To change the error detection severity set the `severity_level` parameter in the behat.yml to the desired level as an int value.
+See `RfcLogLevel` for all possible levels.
+
+Example setting severity level to `RfcLogLevel::NOTICE`:
+```
+contexts:
+  - drunomics\BehatDrupalUtils\Context\DrupalErrorCheckApiContext:
+      severity_level: 5
```

Comment:
Configuration should always be user readable.
we could map strings like 'notice' to internally internally. But in the end  the only meaningful value to configure is notices, so why not simple add a boolean like 
  fail_on_notice: true
instead?